ledger/src/snapshot_utils.rs

```diff
@@ -224,6 +226,7 @@ pub fn bank_from_archive<P: AsRef<Path>>(
         unpacked_accounts_dir,
     )?;
 
+    bank.purge_zero_lamport_accounts();
```

Comment:
As discussed, now we both do purge zero lamport accounts when saving and loading snapshots.

Reply:
In contract to [this](https://github.com/solana-labs/solana/pull/7013/files#r349415024), this call cannot be omitted for security reasons.